src/index.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable import/no-mutable-exports */
+import './style.css';
+
+const submit = document.querySelector('.add-btn');
+const tasksDiv = document.querySelector('.tasks');
+export const taskInput = document.querySelector('.add-task input');
+
+let editId;
+let isEditTask = false;
+let tasks = JSON.parse(localStorage.getItem('task-list'));
+
+export const addElemToPage = () => {
```

Comment:
I would advise you to look out for unused functions, They make your code a bit messy. what do you think?

Reply:
thanks, I'll fix that 👍